src/net.h

```diff
@@ -240,7 +240,7 @@ class CConnman
 
     bool IsMasternodeOrDisconnectRequested(const CService& addr);
 
-    void PushMessage(CNode* pnode, CSerializedNetMsg&& msg, bool allowOptimisticSend = DEFAULT_ALLOW_OPTIMISTIC_SEND);
+    void PushMessage(CNode* pnode, CSerializedNetMsg&& msg);
```

Comment:
Is it really safe to do remove this? https://github.com/dashpay/dash/blob/develop/src/net.h#L42-L54

Reply:
Yepp it's safe as all my testing in the last days has shown. And also, there was only one place left that allowed optimistic send, which was `SendRejectsAndCheckIfBanned`. I removed that comment + DEFAULT_ALLOW_OPTIMISTIC_SEND as well now.